tests/e2e_oss/README.md

```diff
@@ -0,0 +1,65 @@
+# E2E OSS tests
+
+This folder contains E2E test framework code, E2E tests and related materials
+(reference outputs, test data, etc.)
+
```

Comment:
Let us have the following sections in README:
1. Environment setup
2. Adding new model
3. Launch validation

Reply:
I fixed all previous comments not related to readme. I suggest to move next changes of readme in next PR when we will make e2e work in precommit, otherwise I think we could discuss about it forever and wont merge this PR